e2e_tests/tests/cluster/test_workspace_org.py

```diff
@@ -5,6 +5,7 @@
 from determined.common.api import authentication, bindings, certs
 from determined.common.experimental import session
 from tests import config as conf
+#from tests import experiment as exp
```

Comment:
nit: either uncomment or remove

Reply:
I added an experiment-move test so this import is now uncommented / used differently